src/services/logger/DatadogLogger.ts

```diff
@@ -1,28 +1,28 @@
 import { datadogLogs } from '@datadog/browser-logs'
 
-import type { EnvVars } from '@/services/env/Env'
+import type Env from '@/services/env/Env'
 import type { ClientConfigInterface } from '@/store/modules/config/config.types'
 
-type Env = (
-  key: keyof Pick<EnvVars,
-  'KUMA_PRODUCT_NAME'
-  >
-) => string
 export default class DatadogLogger {
-  env: Env
-  constructor(env: Env) {
+  env: Env['var']
+
+  constructor(env: Env['var']) {
     this.env = env
   }
 
   setup(config: ClientConfigInterface) {
     if (config.reports.enabled) {
+      const service = this.env('KUMA_PRODUCT_NAME').toLowerCase().replaceAll(' ', '-') + '-ui'
```

Comment:
**Comment**: I made sure this will end in Datadog as `kuma-ui` so it can be filtered from things related to Kuma itself which might be using the same Datadog instance (or not, still it’s nicer this way).

Reply:
should we use `kuma-gui` not `-ui`. I actually prefer `ui` over `gui` (gooey), but seems like its a pattern we should stick with.

Also, if we are making a change here anyway can we use backticks/templates here instead of addition?

Reply:
In isolation, I would’ve used `-gui`, but I found that UI apps in Datadog were suffixed with `-ui` so I thought it’s better to have the consistency there.